internal/demoproduct/ui/product.css

```
@@ -0,0 +1,74 @@
+.eyebrow {
+  color: #6be4ce;
+  font-size: .75rem;
+  font-weight: 700;
+  letter-spacing: .15em;
+  text-transform: uppercase;
+}
+
+h1 {
+  max-width: 12ch;
+  margin: .4rem 0 1rem;
+  font-family: Iowan Old Style, Palatino Linotype, serif;
```

Stylelint (17.14.0)

[error] Expected quotes around "Iowan Old Style" (font-family-name-quotes)

(font-family-name-quotes)